actions/ql/lib/ext/config/actions_permissions.yml

```diff
@@ -22,16 +22,21 @@ extensions:
       - ["actions/stale", "pull-requests: write"]
       - ["actions/attest-build-provenance", "id-token: write"]
       - ["actions/attest-build-provenance", "attestations: write"]
+      - ["actions/deploy-pages", "pages: write"]
+      - ["actions/deploy-pages", "id-token: write"]
+      - ["actions/delete-package-versions", "packages: write"]
```

Comment:
From [here](https://github.com/actions/delete-package-versions) I see "The token needs the delete packages and read packages scope." Does that mean that we should also add `packages: read`?

Reply:
Thanks for checking! I was under the impression write includes read so `packages: write` will be enough?

Reply:
Ah, I can see [here](https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/controlling-permissions-for-github_token) that you are right.

I am sorry I missed your comment. In the future, feel free to re-request a review (the circle-arrow in the reviewers section), then I will get pinged.